lib/reporters/cli/index.js

```diff
@@ -81,13 +81,80 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
+                // add \n between sets of angled brackets and text node between them
+                .replace(/>\s+(.+?)\s+<(?!\/)/g, '>\n$1\n<')
+                // add \n between angled brackets and text node between them
+                .replace(/>(.+?)<([a-zA-Z])/g, '>\n$1\n<$2')
+                // detect a header of XML
+                .replace(/\?></, '?>\n<');
+            const xmlArr = xmlString.split('\n'); // split it into an array (for analise each line separately
+
+            if ((/^<[?]xml/).test(xmlArr[0])) { start++; } // if the first line is a header, ignore it
+
+            for (i = start; i < xmlArr.length; i++) { // for each line
+                line = xmlArr[i].replace(/^\s+|\s+$/g, ''); // trim it (just in case)
+
+                if ((/^<[/]/).test(line)) { // if the line is a closing tag
+                    tabs = tabs.replace(indent, ''); // remove one indent from the store
+                    xmlArr[i] = tabs + line; // add the tabs at the beginning of the line
+                }
+                else if ((/<.*>.*<\/.*>|<.*[^>]\/>/).test(line)) { // if the line contains an entire node
+                    // leave the store as is
+                    xmlArr[i] = tabs + line; // add the tabs at the beginning of the line
+                }
+                // if the line starts with an opening tag and does not contain an entire node
+                else if ((/<.*>/).test(line)) {
+                    xmlArr[i] = tabs + line; // add the tabs at the beginning of the line
+                    tabs += indent; // and add one indent to the store
+                }
+                else { // if the line contain a text node
+                    xmlArr[i] = tabs + line; // add the tabs at the beginning of the line
+                }
+            }
+
+            // PART 3: return formatted string (source)
+            return xmlArr.join('\n'); // rejoin the array to a string and return it
+        }
+
         // show the body of the response for single request
         if (options.singleRequest &&
             run.executions &&
             run.executions.length &&
             run.executions[0].response &&
             run.executions[0].response.text()) {
-            print(run.executions[0].response.text());
+            if (run.executions[0].response.headers.all()
+                .find((element) => element.key === 'Content-Type') // eslint-disable
+                .value.includes('application/xml')) {
+                const input = run.executions[0].response.text(),
+
+                    prettyxml = formatXML(input);
+
+                print(prettyxml);
+            }
+
+            else if (run.executions[0].response.headers.all()
+                .find((element) => element.key === 'Content-Type') // eslint-disable
+                .value.includes('application/json')) {
+                const a = JSON.parse(run.executions[0].response.text());
+
+                print(JSON.stringify(a, undefined, '\t'));
```

Comment:
Let's use SPACES instead of TAB, so in order to have consistency across the board. We can keep it as 2 or 4 spaces, both are fine.